docs/readmes/lte/debug_dp_probe.md

```diff
@@ -12,27 +12,32 @@ The datapath probe CLI helps an operator to probe the Datapath to the UE.
 
 ```sh
 # On your GW host, run the following command
-$ dp_probe_cli.py -i <IMSI>
-$ dp_probe_cli.py -i 1010000051011
+$ dp_probe_cli.py -i <IMSI> --direction <ingress/egress>
+$ dp_probe_cli.py -i 1010000051011 --direction ingress
```

Comment:
Can you change the arguments, ingress => DL and egress => UL. these are standard LTE terminology.

Reply:
Updated to UL/DL